### ISSUE TITLE
Consider adding an Org or Markdown source for the manual

### ISSUE THREAD

**Issue by shrysr** [Emacs101/howm-manual#3]
Thanks for creating the manual. 

It would be great if you could include the org or markdown source to the manual. This will make it easier for others to contribute, and maybe the PDF's could be auto-generated on a push.

For example, some hacks and workflows were documented on the Emacswiki: https://www.emacswiki.org/emacs/HowmMode

**Comment by Emacs101** [comment]
I will definitely consider it, even though it won’t be an easy task due to the intricate book design and numerous illustrations.

**Comment by shrysr** [comment]
Yeah. I was just re-reading the manual and I see what you mean. I'm happy to help as I can, considering howm is a daily driver. One interesting thing about using a markup format is that the result could also be compiled into a static site in addition to a PDF making it more accessible. If Kazuyuki san agrees, this text could perhaps be included as docs in the howm package or converted into an info like document, which might be cool. 

Some thoughts / ideas:
1. re-design only the complex illustrations such that they are independent images and so can be referenced as usual in org mode or markdown text as images. 
2. Does it make sense to consider 2 documents, where one could be the illustrated guide, and the other could be the base text that forms the base for a manual. By doing so, the illustrations could perhaps be decoupled meaningfully from the text.

**Comment by Emacs101** [comment]
Thank you for the ideas! I think it is doable to create a plain text document in markdown or org format. I agree that some people don't need as many illustrations. I'll try to do that, but not until the end of January.

**Comment by brongulus** [comment]
I would like if atleast the text was included in howm package as an `info` file so that its easily accessible inside of emacs without having to rely on PDF. Even org-mode's own documentation is converted from `.org` to `.info` for use inside of emacs. Something to consider.

**Comment by Emacs101** [comment]
Hi @brongulus 
I considered the idea of creating a folder with TXT or ORG files that users could readily add to their Howm folder. However, implementing this will necessitate extensive rewriting.

I've developed a PDF version tailored specifically for absolute beginners. That's why it contains numerous illustrations and detailed explanations. The objective was to familiarize individuals with Emacs, and I believe Howm aligns perfectly with that goal.

As for the built-in manual, I see it as a separate issue. I might explore it later, but right now, my main focus is updating the manual for Russian-speaking users.

**Comment by stanislavsay** [comment]
Андрей, а возможно ли сделать версию книги в формате epub, чтобы было удобно почитать пока едешь в метро?))) PDF на смартфоне слишком мелко выглядит и место, где остановился не всегда корректно запоминает. Заранее благодарен и спасибо вам за огромный труд и популяризацию заметкоделия!

**Comment by Emacs101** [comment]
@stanislavsay 
Я попробовал. Там, к сожалению, нужно переделывать иллюстрации, иначе форматирование плывет катастрофически. Постараюсь что-то придумать, но, боюсь, Вы к этому моменту уже успеете дочитать существующий вариант :)

**Comment by stanislavsay** [comment]
Жаль, я думал у вас книга в latex или org, тогда через pandoc можно было бы конвернуть. Но и на том спасибо большое!

**Comment by Emacs101** [comment]
@stanislavsay 
Там основная проблема с иллюстрациями. Я думаю перевести текст в Latex, но картинки все равно нужно будет исправлять. Если потом получится конвертировать через Pandoc,  я тут сигнализирую. :)

**Comment by kaorahi** [comment]
epub! (00e6b9d)
